x-pack/plugins/security_solution/public/common/components/event_details/table/investigate_in_timeline_button.tsx

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
I renamed this file, there are (almost) no changes in here

Reply:
💯